wallet/wallet.go

```diff
@@ -3393,8 +3393,31 @@ func (w *Wallet) TotalReceivedForAddr(addr btcutil.Address, minConf int32) (btcu
 // returns the transaction upon success.
 func (w *Wallet) SendOutputs(outputs []*wire.TxOut, keyScope *waddrmgr.KeyScope,
 	account uint32, minconf int32, satPerKb btcutil.Amount,
-	coinSelectionStrategy CoinSelectionStrategy, label string) (
-	*wire.MsgTx, error) {
+	coinSelectionStrategy CoinSelectionStrategy, label string) (*wire.MsgTx,
+	error) {
+
+	return w.sendOutputs(outputs, keyScope, account, minconf, satPerKb,
+		coinSelectionStrategy, label)
+}
+
+// SendOutputsWithInput creates and sends payment transactions using the
+// provided selected utxos It returns the transaction upon success.
+func (w *Wallet) SendOutputsWithInput(outputs []*wire.TxOut,
+	keyScope *waddrmgr.KeyScope,
+	account uint32, minconf int32, satPerKb btcutil.Amount,
+	coinSelectionStrategy CoinSelectionStrategy, label string,
+	selectedUtxos []wire.OutPoint) (*wire.MsgTx, error) {
+
+	return w.sendOutputs(outputs, keyScope, account, minconf, satPerKb,
+		coinSelectionStrategy, label, selectedUtxos...)
+}
+
+// SendOutputs creates and sends payment transactions.It returns the transaction
```

Comment:
nit: need to update the Godoc to lowercase as well. Also, there's a space missing after the full stop.